### DISCUSSION TITLE
0.20.0

### DISCUSSION THREAD

**Discussion by long2ice** [tortoise/tortoise-orm#1451] [Announcements]
### Added
- Allow ForeignKeyField(on_delete=NO_ACTION) (#1393)
- Support `pydantic` 2.0. (#1433)

### Fixed
- Fix foreign key constraint not generated on MSSQL Server. (#1400)
- Fix testcase error with python3.11 (#1308)

### Breaking Changes
- Drop support for `pydantic` 1.x.
- Drop support for `python` 3.7.
- Param `config_class` of `pydantic_model_creator` is renamed to `model_config`.
- Attr `config_class` of `PydanticMeta` is renamed to `model_config`.

<hr /><em>This discussion was created from the release <a href='https://github.com/tortoise/tortoise-orm/releases/tag/0.20.0'>0.20.0</a>.</em>

**Comment by lelecanfora** [comment]
Does the contrib package supports pydantic 2.1? It is really not clear from the changelog.

**Reply by long2ice** [reply]
Which should be for all pydantic >= 2.0
